From c54d86719437156cbaa232f7545d9965302a76bf Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:05:06 -0400
Subject: [usb] yurex: fix spelling mistake in comment

Message-id: <1410887463-20674-235-git-send-email-dzickus@redhat.com>
Patchwork-id: 95542
O-Subject: [RHEL7 PATCH 234/591] USB: yurex: fix spelling mistake in comment
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 3fb4c07a52d7d0bf7d880538c11629e69fc8be2e
Author: Rahul Bedarkar <rahulbedarkar89@gmail.com>
Date:   Mon Dec 30 20:49:19 2013 +0530

    USB: yurex: fix spelling mistake in comment

    fix spelling mistake in comment

    Signed-off-by: Rahul Bedarkar <rahulbedarkar89@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/misc/yurex.c b/drivers/usb/misc/yurex.c
index b6ab515..0609114 100644
--- a/drivers/usb/misc/yurex.c
+++ b/drivers/usb/misc/yurex.c
@@ -464,7 +464,7 @@ static ssize_t yurex_write(struct file *file, const char *user_buffer, size_t co
   goto error;
 
  mutex_lock(&dev->io_mutex);
- if (!dev->interface) {  /* alreaday disconnected */
+ if (!dev->interface) {  /* already disconnected */
   mutex_unlock(&dev->io_mutex);
   retval = -ENODEV;
   goto error;
-- 
1.7.1